ddp-workspace/projects/ddp-rgp/src/app/components/activity/activity.component.html

```diff
@@ -12,7 +12,7 @@
 
   <ng-container *ngIf="isLoaded && model">
     <section class="section">
-      <ddp-subject-panel></ddp-subject-panel>
+      <ddp-subject-panel [invitationId]="invitationId"></ddp-subject-panel>
```

Comment:
@MocanaAtBroad do we need this panel here at all? I'm not sure there is an admin account for rgp. Or do we plan to add it in the future?